src/snowflake/cli/plugins/snowpark/commands.py

```diff
@@ -76,6 +76,8 @@ def deploy(
     All deployed objects use the same artifact which is deployed only once.
     """
     snowpark = cli_context.project_definition
+    database = snowpark.get("database") or options.get("database")
+    schema = snowpark.get("schema") or options.get("schema")
```

Comment:
This has to be specified for every function or procedure:
```yml
procedures:
 - name: foo
   database: bar
   schema: foo
 - name: foo2
   database: bar
   schema: baz
```

Reply:
There's also no need to touch `options` as FQN resolution uses connection details that already include this information.